packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -150,7 +150,7 @@ export default class RequestClient {
     }))
   }
 
-  async #request ({ path, method = 'GET', data, skipPostResponse, signal }) {
+  async request ({ path, method = 'GET', data, skipPostResponse, signal }) {
```

Comment:
We should clarify this is internal method, not meant to be used as public API.
```suggestion
  /** @protected */
  async request ({ path, method = 'GET', data, skipPostResponse, signal }) {
```